Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -556,11 +556,11 @@ def genS(fragment,howMuch):
                            '--datatier':'GEN-SIM-DIGI-RECO,DQMIO',
                            '--relval':'27000,3000'},
                           step1Defaults])
-step1FastUpg2015Defaults =merge([{'-s':'GEN,SIM,RECOBEFMIX,DIGI:pdigi_valid,L1,L1Reco,RECO,EI,HLT:@relval25ns,VALIDATION:@standardValidation,DQM:@standardDQM',
+step1FastUpg2015Defaults =merge([{'-s':'GEN,SIM,RECOBEFMIX,DIGI:pdigi_valid,L1,L1Reco,RECO,EI,DIGI2RAW,HLT:@relval25ns,VALIDATION:@standardValidation,DQM:@standardDQM',
```

Comment:
So far I use what is in 81x unless I missed a commit. Let's see what the tests say



On 16 Apr 2016, at 09:00, Hengne Li <notifications@github.com<mailto:notifications@github.com>> wrote:


In Configuration/PyReleaseValidation/python/relval_steps.py<https://github.com/cms-sw/cmssw/pull/14107#discussion_r59963714>:

> @@ -556,11 +556,11 @@ def genS(fragment,howMuch):
>                             '--datatier':'GEN-SIM-DIGI-RECO,DQMIO',
>                             '--relval':'27000,3000'},
>                            step1Defaults])
> -step1FastUpg2015Defaults =merge([{'-s':'GEN,SIM,RECOBEFMIX,DIGI:pdigi_valid,L1,L1Reco,RECO,EI,HLT:@relval25ns,VALIDATION:@standardValidation,DQM:@standardDQM',
> +step1FastUpg2015Defaults =merge([{'-s':'GEN,SIM,RECOBEFMIX,DIGI:pdigi_valid,L1,L1Reco,RECO,EI,DIGI2RAW,HLT:@relval25ns,VALIDATION:@standardValidation,DQM:@standardDQM',


@davidlange6<https://github.com/davidlange6> why we don't change the HLT:@relval25ns here for fastsim also to HLT:@relval2016 ?

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/14107/files/db3d50025447fb8c16dff5ab467a0b1d7a3add5a#r59963714>